mpo.rga.Dayon.yml

```diff
@@ -0,0 +1,61 @@
+app-id: mpo.rga.Dayon
+runtime: org.gnome.Platform
```

Comment:
do you really need the GNOME runtime?

Reply:
It's a java awt/swing application which runs under gnome.

Reply:
GNOME or just gtk? the latter (gtk3) is in the freedesktop-sdk

Reply:
You're right, it works as well with freedesktop-sdk!